Moved OAK-SOM-IOT to other devices, combined OAK-SOM-PROs into single… (#270)
luxonis/depthai-hardware: docs/source/pages/BW2099.rst

`Buy it on Luxonis shop <https://shop.luxonis.com/collections/system-on-module-som/products/oak-som-pro-5-pcs>`__

.. figure:: /_static/images/SOM-PRO/som-pros-front.jpg

    **left**: OAK-SoM-Pro Keem Bay, **right**: OAK-SoM-Pro Myriad X

Overview
********

> The **OAK-SoM-Pro** is a System on Module (SoM) designed for integration into top-level system with

Reviewer: ```suggestion
The **OAK-SoM-Pro** is a System on Module (SoM) designed for integration into top-level systems with
```

> a need for a low-power, high performance, real-time AI and depth perception. To be used in industrial automation, robotics and security systems.

Reviewer: ```suggestion
a need for a low-power, high performance, real-time AI, and depth perception. It is ideal for use in industrial automation, robotics, and security systems.
```
